docs/index.rst

```diff
@@ -0,0 +1,155 @@
+.. |config1| image:: images/config_1.png
+.. |config2| image:: images/config_2.png
+
+
+Person Detection Plugin
+=======================
+
+The *fledge-south-person-detection* detects a person on a live video feed from either a camera or on a network
+stream. It uses google's mobile net ssd version 2 to detect a person. The bounding boxes and confidence scores are displayed on the same video frame itself.
+Also FPS (Frames per second are also displayed on the same frame.) The detection results are also converted into readings. The readings have mainly three things:
+
+1. *Count* : No of people detected.
+
+2. *Coordinates* : the coordinates of the bounding box detected (2 points each having x and y respectively)
```

Comment:
May be say (x,y) coordinates of top-left and bottom right corners of bounding box for each detected person.